spmd/tensor/parallel/api.py

```diff
@@ -278,19 +284,19 @@ def _parallelize_multihead_attn(
 
     Args:
         module (nn.Module):
```

Comment:
can we change the all the parameter comment, i.e. :class:`nn.Module` inside the "(:class:`nn.Module`) instead? that would match more to the pytorch comment style.

Reply:
sure. Let me do that.